src/bundle/Resources/views/themes/standard/design_system/components/Badge.html.twig

```diff
@@ -0,0 +1,29 @@
+{% set badge_classes =
+    html_cva(
+        base: html_classes(
+            'ids-badge',
+            {
+                'ids-badge--expanded': is_expanded
+            }
+        ),
+        variants: {
+            size: {
+                small: 'ids-badge--small'
+            }
+        }
+    )
+%}
+
+<div
+    class="{{
```

Comment:
you can rebase with main where I disabled twig prettier and change this formatting to sth "normal" :sweat_smile: